conformance/src/main/scala/za/co/absa/enceladus/conformance/DynamicConformanceJob.scala

```diff
@@ -108,6 +108,14 @@ object DynamicConformanceJob {
     enabled
   }
 
+  private def isAutocleanStdFolderEnabled()(implicit cmd: CmdConfig): Boolean = {
+    val enabled = getCmdOrConfigBoolean(cmd.autocleanStandardizedFolder,
+      "conformance.autoclean.standardized.hdfs.folder",
+      defaultValue = false)
+    log.info(s"Autoclean standardized HDFS folder = $enabled")
```

Comment:
Also I find it weird to log this value upon querying the function. Imagine it would be queried twice for some reason.
IMHO the logging would be better suited in the creation of `CmdConfig` (Actually why not to log all switches there? 🤔

Reply:
This is done for consistency with `isExperimentalRuleEnabled()` and `isCatalystWorkaroundEnabled()`.
Maybe we can demote these defs to lazy vals as a part of refactoring (some other task), that way we can [potentially] save both query time and logging it twice.

Reply:
OK, let's have it this way now. In refactoring having a Configuration object - like you have in Migration nad Continuous migration - would be a nice to group the setup into.